lib/shipit.rb

```diff
@@ -206,6 +207,16 @@ def default_inactivity_timeout
     secrets.commands_inactivity_timeout || 5.minutes.to_i
   end
 
+  def automatically_prepend_bundle_exec
+    @automatically_prepend_bundle_exec ||= begin
```

Comment:
That won't work. If you set the variable to false, the `||=` will set it back to `true` on the next invocation.

You need to use `unless defined?(@automatically_prepend_bundle_exec)`